_locales/pl/messages.json

```diff
@@ -208,7 +208,7 @@
 		"description": "Options page label: 'characters'"
 	},
 	"optionFilenameReplacementCharacter": {
-		"message": "replacement character",
+		"message": "znak zastępczy",
 		"description": "Options page label: 'replacement character'"
```

Comment:
why not here ?

Reply:
`description` is only a hint for translators. Translating it is pointless.